src/TrackerCouncil.Smz3.UI/TrackerCouncil.Smz3.UI.csproj

```diff
@@ -6,7 +6,7 @@
         <BuiltInComInteropSupport>true</BuiltInComInteropSupport>
         <ApplicationManifest>app.manifest</ApplicationManifest>
         <AvaloniaUseCompiledBindingsByDefault>true</AvaloniaUseCompiledBindingsByDefault>
-        <Version>9.9.6</Version>
+        <Version>9.9.7</Version>
```

Comment:
You remembered!

Reply:
Shout-out to finally being able to run the tests in Linux!